src/Fields/DateRangePicker.php

```diff
@@ -75,6 +76,18 @@ public function disableClear(bool|Closure $disable = true) : static
         return $this;
     }
 
+    public function icon(string|Closure|null $icon = null): static
+    {
+        $this->icon = $icon;
+
+        return $this;
+    }
+
+    public function getIcon(): string
+    {
+        return $this->evaluate($this->icon) ?? 'heroicon-m-calendar-days';
```

Comment:
Can we set the default icon to the property?

Reply:
I could do that for the picker. However, if I don't set an icon on the filter, the icon on the picker would be nulled.

Calling `DateRangeField::make()`would hand down a `null` icon to `DateRangePicker` in `src/Filters/DateRangeFilter.php:L196`.

But then I would also have to set the default icon for the filter, too. And then we would have the default icon on the picker and on the filter. That would be duplicate code again.

Reply:
Understood